subjects/get-json/README.md

```diff
@@ -5,15 +5,15 @@
 In this exercise, we will focus on building complex async flows with promises.
 
 Create a `getJSON` function that takes 2 parameters:
-- `path`, that will be the url called by your function
-- `params` *optional*, that will be the search parameters appended to your url
+- `path`, string that will be the url called by your function
+- `params` *optional*, object that will be the search parameters appended to your url
 
 `getJSON` must construct a valid url with the `path` and stringified `params`
 and call `fetch` with it.
-If the response is not ok, your function must throw an error using
-the response status message.
+If the response is not ok (i.e. response status is not 2xx), your function must throw an error using
```

Comment:
I was being evasive on purpose here, so student search a bit.
Also it was a hint on [response/ok](https://devdocs.io/dom/response/ok)

Which is cleaner to use than checking the range of `statusCode`.

I'm pretty sure everyone will just check status code directly if we write that.